libs/core/src/components/pds-table/pds-table.scss

```diff
@@ -76,7 +76,7 @@
   top: 0;
   transition: opacity 0.1s ease;
   width: 4px;
-  z-index: 10;
```

Comment:
Will keep an eye out for this one when reviewing in the monolith to be sure - overall low but higher than most lows

Reply:
Will try converting to token and test as well.